runtime/coreclr-bridge.m

```diff
@@ -47,4 +47,17 @@
 	return rv == 0;
 }
 
+void
+xamarin_handle_bridge_exception (GCHandle gchandle, const char *method)
+{
+	if (gchandle == INVALID_GCHANDLE)
+		return;
+
+	if (method == NULL)
+		method = "<unknown method";
```

Comment:
Did you mean this to be `"<unknown method>"`?

Reply:
Yes, I did.